spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -53,6 +54,55 @@ definitions:
             type: s32
             units: 0.1 mm
             desc: Phase bias for specified signal
+ - PackedOsrContent:
```

Comment:
I'd move these out of SSR and into Observations. It does have strong affinity to observations and the structure sharing is cumbersome currently.